src/components/atoms/faq/FaqItem.jsx

```diff
@@ -0,0 +1,23 @@
+import { useState } from 'react';
+import { FaqItemContainer, Answer, ArrowIcon } from './style';
+import ArrowDropDownIcon from '@mui/icons-material/ArrowDropDown';
```

Comment:
Use icons from iconify here that's why it's giving an error when uninstalling these.